software/SolarAISVN/SolarAISVN.ino

```diff
@@ -93,8 +100,9 @@ void makeIFTTTRequest() {
 
   // raw and converted voltage reading
   adcValue = analogRead( 34 );
-  String jsonObject = String("{\"value1\":\"") + adcValue + "\",\"value2\":\"" + (adcValue * 2.4)
-                      + "\",\"value3\":\"" + millis() + "\"}";
+  batValue = analogRead( 35 );
+  String jsonObject = String("{\"value1\":\"") + adcValue + "\",\"value2\":\"" + int((adcValue * 0.826 + 150) * 3)
+                      + "\",\"value3\":\"" + int((batValue * 0.826 + 150) * 2) + "\"}";
```

Comment:
Submit the battery value as well.

Reply:
Please relocate the calculation of the battery voltage value to a different location.